test/integration/security-rules.spec.ts

```diff
@@ -137,7 +152,7 @@ describe('admin.securityRules', () => {
       return admin.securityRules().releaseFirestoreRuleset(oldRuleset);
     }
 
-    after(() => {
+    afterEach(() => {
```

Comment:
This change is not really needed right. I'd say lets either keep it as `after` or lets rename the helper to something like `revertFirestoreRulesetIfModified` so the logic of the test suite becomes clearer.

Reply:
I like keeping it as an afterEach. (It's not an issue here, but there are ordering implications to mixing after and afterEach which I tripped on elsewhere in this PR.) Therefore I've renamed the helper to revertFirestoreRulesetIfModified.